hoomd/mpcd/collide.py

```diff
@@ -124,12 +126,11 @@ class CollisionMethod(Operation):
             :class:`~hoomd.md.methods.ConstantVolume` with no thermostat (NVE).
             It is generally **not** a good idea to use a thermostat because the
             MPCD particles themselves already act as a heat bath for the
-            embedded particles.
+            embedded particles. If using rigid bodies, thermostat is required.
 
             Warning:
-                Do not embed particles that are part of a rigid body. Momentum
-                will not be correctly transferred to the body. Support for this
-                is planned in future.
+                Embedding particles that are part of a rigid body is not available
+                when using domain decomposition. Support for this is planned in future.
```

Comment:
Don't you also need to document about setting the masses of the particles, and which particles should be embedded (constituents not center)? Probably a full paragraph description is needed, and you can mention that setting a temperature `kT` is required with rigid bodies at that point, in addition to warning that this will not work.